tests/query_params_test.go

```diff
@@ -0,0 +1,216 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"bytes"
+	"context"
+	"go-safeweb/testing/requesttesting"
+	"net/http"
+	"testing"
+)
+
+const statusOK = "HTTP/1.1 200 OK\r\n"
+const statusBadReq = "HTTP/1.1 400 Bad Request\r\n"
+
+// isOkReq is a helper function that checks whether the server response is 200
+func isOkReq(resp []byte) bool {
+	if bytes.HasPrefix(resp, []byte(statusOK)) {
+		return true
+	}
+	return false
```

Comment:
Simple conditions can be returned directly like this:

return bytes.HasPrefix(resp, statusOK)

Reply:
Resolved in 12a32a